extensions/Worker.Extensions.SendGrid/Worker.Extensions.SendGrid.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.SendGrid</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.SendGrid</RootNamespace>
+    <Description>Azure Service Bus extensions for .NET isolated functions</Description>
+
+    <!--Version information-->
+    <VersionPrefix>3.0.2</VersionPrefix>
+
+    <!--Temporarily opting out of documentation. Pending documentation-->
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>

```

Comment:
If all the docs are done, remove this.

Reply:
Copied the flag from other extension. Not sure which doc is being referenced here.

Reply:
This is about the XML docs applied to the code. Looks like the attribute is documented, so this should be removed.